integration/raft/channel_participation_test.go

```diff
@@ -123,59 +126,99 @@ var _ = Describe("ChannelParticipation", func() {
 				Expect(resp.Status).To(Equal(common.Status_SUCCESS))
 				expectedBlockNumPerChannel := map[string]int{"participation-trophy": int(expectedChannelInfoPT.Height - 1)}
-				assertBlockReception(expectedBlockNumPerChannel, orderers, peer, network)
+				assertBlockReception(expectedBlockNumPerChannel, members, peer, network)
 
 				By("checking the channel height")
-				channelparticipation.ListOne(network, o, expectedChannelInfoPT)
+				channelInfo := channelparticipation.ListOne(network, o, "participation-trophy")
+				Expect(channelInfo).To(Equal(expectedChannelInfoPT))
+			}
+
+			By("joining orderer3 to the channel as a follower")
+			// make sure we can join using a config block from one of the other orderers
+			configBlockPT := nwo.GetConfigBlock(network, peer, orderer2, "participation-trophy")
+			expectedChannelInfoPTFollower := channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "onboarding",
+				ClusterRelation: "follower",
+				Height:          0,
 			}
+			channelparticipation.Join(network, orderer3, "participation-trophy", configBlockPT, expectedChannelInfoPTFollower)
+
+			By("ensuring orderer3 completes onboarding successfully")
+			expectedChannelInfoPTFollower.Status = "active"
+			expectedChannelInfoPTFollower.Height = expectedChannelInfoPT.Height
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfoPTFollower))
+
+			By("adding orderer3 to the consenters set")
+			channelConfig := nwo.GetConfig(network, peer, orderer1, "participation-trophy")
+			c := configtx.New(channelConfig)
+			err := c.Orderer().AddConsenter(consenterChannelConfig(network, orderer3))
+			Expect(err).NotTo(HaveOccurred())
+			computeSignSubmitConfigUpdate(network, orderer1, peer, c, "participation-trophy")
+
+			By("ensuring orderer3 transitions from follower to member")
+			// config update above added a block
+			expectedChannelInfoPT.Height++
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfoPT))
+
+			// make sure orderer3 finds the leader
+			newLeader := findLeader([]*ginkgomon.Runner{ordererRunners[2]})
+			Expect(newLeader).To(Equal(leader))
+
+			By("submitting transaction to orderer3 to ensure it is active")
+			env = CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello-again"))
+			expectedChannelInfoPT.Height++
+			resp, err := ordererclient.Broadcast(network, orderer3, env)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
+			expectedBlockNumPerChannel := map[string]int{"participation-trophy": int(expectedChannelInfoPT.Height - 1)}
```

Comment:
I _think_ this number should be entirely predictable.  If so, I'd much rather see us using literals than storing it in a value which is mutated throughout the test.  If we change behavior, such that the block number is now wrong, then we want our test to catch the the behavior changed.  If the behavior changes and we understand and accept that, it's easy enough to change a literal when the test fails.